### ISSUE TITLE
Comma between moves prevents interaction

### ISSUE THREAD

**Issue by sphaso** [neagle/gokibitz#96]
Hello!
I've just noticed that:
- commenting ("user" comment, not game comment) with a list of comma-separated moves
- when hovering over each coordinate, the latter is not displayed on the board

E.g. https://gokibitz.com/kifu/NJ37JY0N?path=139

I realize this is a bit of a nitpick and I'm not sure how you feel about it. I think it would be nice to have a way to propose a list of alternative moves (so not a sequence) and still have interactivity.

**Comment by neagle** [comment]
Yeah, I've noticed this before, too. The problem is that when I created the sequences feature, it allowed moves to be either space-separated or comma-separated, because I've seen people write sequences that way: "B5, C6, H14", etc. But it's possible we could change that, since the in-game sequence creator seems to have been popular enough that I think every time I see an intended sequence, it's space-separated.
